runtime/doc/lua.txt

```diff
@@ -1302,38 +1392,37 @@ Option:get()
             print("J is enabled!")
         end
 <
-                                                            *vim.opt:append()*
-Option:append(value)
-
-    Append a value to string-style options. See |:set+=|
 
-    These are equivalent: >lua
-        vim.opt.formatoptions:append('j')
-        vim.opt.formatoptions = vim.opt.formatoptions + 'j'
-<
-                                                           *vim.opt:prepend()*
-Option:prepend(value)
+    Return: ~
+        string|integer|boolean|nil value of option
 
+Option:prepend({value})                                    *vim.opt:prepend()*
     Prepend a value to string-style options. See |:set^=|
 
     These are equivalent: >lua
         vim.opt.wildignore:prepend('*.o')
         vim.opt.wildignore = vim.opt.wildignore ^ '*.o'
 <
-                                                            *vim.opt:remove()*
-Option:remove(value)
 
+    Parameters: ~
+      • {value}  (string) Value to prepend
+
+Option:remove({value})                                      *vim.opt:remove()*
     Remove a value from string-style options. See |:set-=|
 
     These are equivalent: >lua
         vim.opt.wildignore:remove('*.pyc')
         vim.opt.wildignore = vim.opt.wildignore - '*.pyc'
 <
 
+    Parameters: ~
+      • {value}  (string) Value to remove
+
+
 ==============================================================================
 Lua module: vim                                                      *lua-vim*
 
-cmd({command})                                                     *vim.cmd()*
+vim.cmd({command})                                                 *vim.cmd()*
```

Comment:
This is fine IMO if it's hard to avoid. It might even be an improvement.

The only case I'm worried about is when `vim.*` is not actually the correct namespace.

Reply:
imo it's an improvement if the full paths are shown
an example would be the treesitter function where I always had to scroll up to find the full function signature
but that might be exactly one of the examples where `vim.x` isn't the full path